staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -100,6 +100,20 @@ func (a *AdmissionOptions) ApplyTo(
 	scheme *runtime.Scheme,
 	pluginInitializers ...admission.PluginInitializer,
 ) error {
+	if a == nil {
+		return nil
```

Comment:
why not to return an error, indicating that it cannot fulfil the method when the reference to `AdmissionOptions` is nil ?

Reply:
This is copied from may other options.ApplyTo.

Reply:
This is our usual pattern for the ApplyTo funcs. You can opt-out of using the options by setting them the nil. We still call the corresponding ApplyTo (which just returns).

Reply:
oh, I see.